test/powershell/Modules/Microsoft.PowerShell.Core/CompatiblePSEditions.Module.Tests.ps1

```diff
@@ -390,7 +390,9 @@ Describe "Import-Module from CompatiblePSEditions-checked paths" -Tag "CI" {
     }
 }
 
-Describe "Additional tests for Import-Module with WinCompat" -Tag "CI" {
+Describe "Additional tests for Import-Module with WinCompat" -Tag "Feature" {
+    $PSDefaultParameterValues = @{ 'It:Skip' = (-not $IsWindows) }
```

Comment:
If you look at other test scripts, the convention is to put it in `BeforeAll` and restore in `AfterAll`

Reply:
This is already fixed. Same thing was advised by @adityapatwardhan .